.github/workflows/dotnet.yml

```diff
@@ -1,8 +1,7 @@
 name: VS4Mac Extension
 env:
-  IDE_TOOLS_RELEASE_VERSION: 1.5.0
-  MEADOW_OS_VERSION: 1.5.0.0
-  VS_MAC_2019_VERSION: 8.10
+  IDE_TOOLS_RELEASE_VERSION: 2.0.0
+  MEADOW_OS_VERSION: 1.8.0.0
```

Comment:
This will be 2.0.0.0 when we release on tuesday

Reply:
Updated!